edi_sale_ubl_oca/views/sale_order.xml

```diff
@@ -26,9 +26,9 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="edi_sale_oca.view_order_form" />
         <field name="arch" type="xml">
-            <field name="origin_exchange_type_id" position="after">
-                <field name="edi_state_id" optional="hide" readonly="1" />
-            </field>
+            <group name="edi_main" position="inside">
```

Comment:
TBH I don't know why it was not working as it used to (beside the `optional="hide"` attr that was there by mistake)